REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.4
 Nettle 0.2
 JSON 0.5
-ZMQ 0.3.1
+ZMQ 0.3.4
```

Comment:
Ah, that makes sense, there was no `unsafe_string(::Message)` method before ZMQ.jl 0.3.3? So ZMQ.jl 0.3.3 is the actual minimum.